2021/ashok2022re/openreview/packages.tex

```diff
@@ -0,0 +1,34 @@
+% Import packages here
+\setcounter{secnumdepth}{3} % default value for 'report' class is "2"
+
+% \usepackage[preprint]{neurips_2019}
+
+\usepackage[utf8]{inputenc} % allow utf-8 input
+\usepackage[T1]{fontenc}    % use 8-bit T1 fonts
+\usepackage{hyperref}       % hyperlinks
+\usepackage{url}            % simple URL typesetting
+\usepackage{booktabs}       % professional-quality tables
+\usepackage{amsfonts}       % blackboard math symbols
+\usepackage{nicefrac}       % compact symbols for 1/2, etc.
+\usepackage{microtype}      % microtypography
+\usepackage{makecell}
+\usepackage{textcomp}
+\usepackage{graphicx}
+\usepackage{multirow}
+\usepackage{float}
+
+\usepackage[dvipsnames]{xcolor}
```

Comment:
Please remove this import (stated in README)

Reply:
Done!